.github/workflows/.combine-dependabot-prs.yml

```
@@ -0,0 +1,24 @@
+jobs:
+  combine-prs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: combine-prs
+        id: combine-prs
+        uses: github/combine-prs@v5.2.0
+        with:
+          labels:
+            dependencies
+            combined
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)